packages/calcite-components/src/components/segmented-control/segmented-control.e2e.ts

```diff
@@ -284,14 +336,18 @@ describe("calcite-segmented-control", () => {
     expect(element).toEqualAttribute("width", "full");
   });
 
-  it("renders default props", async () => {
+  it("inheritable props: `appearance`, `layout`, and `scale` modified on the parent get passed to items", async () => {
     const page = await newE2EPage();
-    await page.setContent("<calcite-segmented-control></calcite-segmented-control>");
-    const element = await page.find("calcite-segmented-control");
-    expect(element).toEqualAttribute("scale", "m");
-    expect(element).toEqualAttribute("layout", "horizontal");
-    expect(element).toEqualAttribute("appearance", "solid");
-    expect(element).toEqualAttribute("width", "auto");
+    await page.setContent(html`
+      <calcite-segmented-control appearance="outline" layout="vertical" scale="l"></calcite-segmented-control>
+    `);
+    const segmentedControlItems = await page.findAll("calcite-segmented-control-item");
+
+    segmentedControlItems.forEach(async (item) => {
```

Comment:
You'll need to use a `for...of` loop since `forEach` doesn't play well w/ `await`.